doc/user/content/ops/monitoring.md

```diff
@@ -140,6 +146,17 @@ Materialize broadly publishes the following types of data there:
   averaged over 10 second windows.
 - Standard process metrics with a `process_*` prefix. For exmple, `process_cpu`.
 
+## System Status Views
+
+{{< warning >}}
+The `mz_catalog` views not part of Materialize's stable interface.
+Backwards-incompatible changes to the exposed views may be made at any time.
+{{< /warning >}}
```

Comment:
I think we can drop this warning because it's covered by the authoritative "system catalog" doc.